lib/metadata/linux/LinuxUtils.rb

```diff
@@ -115,5 +115,24 @@ def self.parse_chkconfig_list(lines)
 
       return ret
     end
+
+    def self.parse_systemctl_list(lines)
+      return [] if lines.nil? || lines.empty?
+
+      lines.each_line.map do |line|
+        line = line.chomp
+        parts = line.split(' ')
+        next unless /^.*?\.service$/ =~ parts[0]
+
+        name, _ = parts[0].split('.')
+
+        # TODO(lsmola) investigate adding systemd targets, which are used instead of runlevels. Drawback, it's not
+        # returned by any command, so we would have to parse the dir structure of /etc/systemd/system/
+        # There is already MiqLinux::Systemd.new(@systemFs) called from class MIQExtract, we should leverage that
+        {:name => name, :systemd_load => parts[1], :systemd_active => parts[2], :systemd_sub => parts[3],
+         :typename => 'linux_systemd', :description => parts[4..-1].join(" "), :enable_run_level => nil,
+         :disable_run_level => nil, :running => parts[3] == 'running' }
```

Comment:
yeah I was used to do this, then I saw the code here uses more like tar.gz style, so I followed that :-)

done